lib/features/details_screen/details_page.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:flutter/cupertino.dart';
```

Comment:
If you want to build app on other devices than iOS, you should use 'package:flutter/material.dart' or 'package:flutter/widgets.dart'